.github/workflows/release-docker-image-server-only.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release UID2 Examples Image for Server Only
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish Pre-release' || format('Release {0}', inputs.release_type)}} UID2 Examples Image for Server Only by @${{ github.actor }}  
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: 'The type of release'
```

Comment:
No need for quotes, and do we need `required: true`?

Reply:
Because it's a type: choice, I don't think we need to add required: true as it's a dropdown menu. I will remove the single quote tho!